tests/phpunit/tests/theme/wpEnqueueStoredStyles.php

```diff
@@ -0,0 +1,74 @@
+<?php
```

Comment:
This change for adding the test class and relocating its tests to it is ready for commit 👍 

I'll commit this separately from the improvement changes in `wpGetGLobalStylesheet.php`.

Reply:
Created https://github.com/WordPress/wordpress-develop/pull/4252 as a confidence check for commit.

Reply:
Committed via https://core.trac.wordpress.org/changeset/55567.

Reply:
Done ✅